### PR TITLE
Adding fields for AQT Sensor

### PR DESCRIPTION
**Pull request by darshanjain12**
Fieldnames for AQT Sensor installed on Terrace floor for outside air properties

### CODE REVIEW

File: ontology/yaml/resources/fields/telemetry_fields.yaml [hunk 1]
```diff
@@ -181,6 +181,15 @@ literals:
 - zone_air_relative_humidity_sensor
 - zone_air_deadband_temperature_setpoint
 - zone_air_temperature_sensor
+- outside_air_co2_concentration_sensor
+- outside_air_co_concentration_sensor
+- outside_air_so2_concentration_sensor
+- outside_air_no2_concentration_sensor
+- outside_air_no_concentration_sensor
+- outside_air_h2s_concentration_sensor
+- outside_air_ozone_level_sensor
```
Comment (tasodorff) [inline ontology/yaml/resources/fields/telemetry_fields.yaml:190]: Ozone measured in meters? `level` inherits from distance, not concentration or density. Please correct.

**Review by tasodorff — DISMISSED** [review]

File: ontology/yaml/resources/fields/telemetry_fields.yaml (hunk 1 above)
Comment (darshanjain12) [reply ontology/yaml/resources/fields/telemetry_fields.yaml]: Yes Thanks

**Comment by cstirdivant** [comment]
@darshanjain12 please fix validator errors

**Comment by darshanjain12** [comment]
> @darshanjain12 please fix validator errors

Done

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**